neqo-transport/src/cc/new_reno_cc.rs

```diff
@@ -286,6 +235,83 @@ impl CongestionControl {
         );
     }
 
+    fn start_pacer(&mut self, now: Instant) {
+        // Start the pacer with a small burst size.
+        self.pacer = Some(Pacer::new(
+            now,
+            MAX_DATAGRAM_SIZE * PACING_BURST_SIZE,
+            MAX_DATAGRAM_SIZE,
+        ));
+    }
+
+    fn next_paced(&self, rtt: Duration) -> Option<Instant> {
+        // Only pace if there are bytes in flight.
+        if self.bytes_in_flight > 0 {
+            Some(
+                self.pacer
+                    .as_ref()
+                    .unwrap()
+                    .next(rtt, self.congestion_window),
+            )
+        } else {
+            None
+        }
+    }
```

Comment:
I think that ultimately we want the pacer to be separate from the CC.  But that can be done later.